virttest/qemu_storage.py

```diff
@@ -385,15 +385,25 @@ def snapshot_apply(self):
 
         decode_to_text(process.system_output(cmd))
 
-    def remove(self):
+    def remove(self, backup_remove=False):
         """
         Remove an image file.
-        """
-        logging.debug("Removing image file %s", self.image_filename)
-        if os.path.exists(self.image_filename):
-            os.unlink(self.image_filename)
+        backup_remove: if set to True, only removes backup copies of
+        image_filename
+        """
+        if backup_remove:
+            logging.debug("Removing image backups of %s", self.image_filename)
+            backup_dir = os.path.join(
```

Comment:
Please use 'utils_misc.get_path' to get 'backup_dir',  get_path function support  absolute path.

Thanks,
Xu

Reply:
Thanks. I have incorporated it now.